vendor/github.com/moby/sys/user/lookup_unix.go

```diff
@@ -149,9 +149,13 @@ func CurrentUserSubGIDs() ([]SubID, error) {
 }
 
 func CurrentProcessUIDMap() ([]IDMap, error) {
+	// We don't need to use /proc/thread-self here because uid_map is a
```

Comment:
We should not modify files in the vendor dir, we should update it from `go.mod`.
But I wonder why the lint doesn't check this now?

Reply:
Ah sorry, this was done automatically by the rebase, I forgot that we moved this to moby. I'll fix it up.